ovos_config/config.py

```diff
@@ -552,3 +552,73 @@ def clear_cache(message=None):
         """DEPRECATED - there is no cache anymore """
         Configuration.updated(message)
 
+
+class ReadOnlyConfig(LocalConf):
+    """ read only  """
+
+    def __init__(self, path, allow_overwrite=False):
+        super().__init__(path)
+        self.allow_overwrite = allow_overwrite
```

Comment:
Should RO status be mutable after init or for any `ReadOnlyConfig` object at all?

Reply:
a verification step should be able to enable this, how this authentication is done is outside the scope of this package but i think it should be mutable no?

Reply:
I was thinking a  `ReadOnlyConfig` is always RO, otherwise it would just be a `LocalConfig`.. i.e. if I pass a skill or plugin a `ReadOnlyConfig`, I'd expect that wouldn't be able to write any changes

Reply:
the reload method will throw exceptions due to the overriding all the dunder methods from dict class, so this need to stay even if private. i think this was the initial reason for it

Reply:
regardless this needs to be compatible with ovos_utils so cant rename the variable and need to keep it.... we can revisit later however, open an issue so we dont forget once this PR is merged